css/css-text/letter-spacing/META.yml

```diff
@@ -0,0 +1,32 @@
+links:
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=195275
```

Comment:
This bug appears to be marked RESOLVED FIXED. Did your script check the status of the WebKit bug?

Reply:
No, it doesn't check that still. 
In this specific case, that bug should be re-opened. I just did it.

Reply:
SGTM. I imagine in the future we will do passes over metadata and look for closed bugs (for any product), so there's not a huge need to do that now.

Reply:
Yes. I would also like to improve this script to the point where its able to update this metadata info with new info comming from the WebKit layout test expectations (like: bug number changed or test not longer marked as failing). Ideally to have it running on a daily/weekly basis to keep the WPT metada bug info updated with any new changes coming from WebKit's TestExpectations info.